src/routes/post.js

```diff
@@ -13,5 +14,6 @@ router.post('/', validateAuthentication, create)
 router.get('/', validateAuthentication, getAll)
 router.post('/comment', validateAuthentication, createComment)
 router.get('/comment', validateAuthentication, findAllComments)
+router.delete('/post-delete/:id', validateAuthentication, deletePost)
```

Comment:
Please change this to `/posts/:id` follow a conventional RESTful endpoint

Reply:
Changed.